flexget/plugins/input/couchpotato.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import requests
+
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+log = logging.getLogger('couchpotato')
+
+
+class CouchPotato(object):
```

Comment:
Usage info should go here as a docstring. This is what's printed when the --doc option is used from the command line.